style.css

```diff
@@ -0,0 +1,278 @@
+/* CSS Fiile for Davins Portfolio */
+
+
+/* -------------------------------------------------------------
+	Generic Body Stuff 
+---------------------------------------------------------------- */
+
+* {
+	box-sizing: border-box;
+}
+
+body {
+	padding: 0;
+	margin: 0;
+	background-color:  hsl(204, 46%, 20%);
+	font-family: Verdana;
+	font-size: 90%;
+	
+}
+
+
+/* ---------------------------------------------------------------------------
+	Generic Classes for  Containers & Sections and Cursive Font for Sections
+---------------------------------------------------------------------------- */
+
+.mainContainerGrid {
+	/* nothing required here, only MQ later */
+}
+
+@media screen and (min-width: 1080px) {
+	.mainContainerGrid {
+		width: 80%;
+		margin: 0 auto;
+	}
+}
+
+.colorBackgroundDark {
+	/* darker stone blue - using HSL so can easily relate color-lightness */
+	background-color:  hsl(204, 46%, 44%);
```

Comment:
again, hex codes for colors everywhere, except when doing opacity, where rgba is okay